extras/prometheus/gmp/README.md

```diff
@@ -0,0 +1,8 @@
+# GMP on GKE cluster
+
+This directory contains files related to configuring probing of Bank of Anthos' microservices with GMP on GKE
+
+* `blackbox-exporter.yaml` - Deployment manifest of [Blackbox exporter](https://github.com/prometheus/blackbox_exporter/)
+* `probes.yaml` - File contains GMP PodMonitoring custom resource which scrape metrics from Blackbox Exporter with configuration for each microservice.
```

Comment:
```suggestion
* `probes.yaml` - contains the GMP PodMonitoring custom resource definitions per microservice. One monitoring configuration per microservice is defined to be run by the  Blackbox exporter.
```

Reply:
Stay consistent with the naming of `Blackbox exporter`. Uppercase B and lowercase e